src/main/java/com/rakovets/course/java/core/practice/date_and_time/DateWrapper.java

```diff
@@ -0,0 +1,74 @@
+package com.rakovets.course.java.core.practice.date_and_time;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAdjuster;
+import java.time.temporal.TemporalAdjusters;
+import java.util.Calendar;
+import java.util.Date;
+
+import static java.util.Locale.US;
+
+public class DateWrapper {
+
+    public static LocalDate getLocalDate(int year, int month, int day) {
+        return LocalDate.of(year, month, day);
+    }
+
+    public static LocalDate addMonths(LocalDate localDate, int months) {
+        return localDate.plusMonths(months);
+    }
+
+    public static String localDatePattern(LocalDate localDate, String pattern) {
+        return localDate.format(DateTimeFormatter.ofPattern(pattern, US));
+    }
+
+    public static String localDateFormat(String localDate, String pattern) {
+        return LocalDate.parse(localDate).format(DateTimeFormatter.ofPattern(pattern, US));
+    }
```

Comment:
Specification violation
```suggestion
    public static LocalDate localDateFormat(String localDate, String pattern) {
        return LocalDate.parse(localDate, DateTimeFormatter.ofPattern(pattern, US));
    }
```